src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/MarkDuplicatesSparkIntegrationTest.java

```diff
@@ -175,6 +174,308 @@ public void testMarkDuplicatesSparkIntegrationTestLocal(
         }
     }
 
+    @Test( dataProvider = "md")
+    public void testMarkDuplicatesSparkMarkingOpticalDuplicatesWithTagging(
+            final File input, final long totalExpected, final long dupsExpected,
+            Map<String, List<String>> metricsExpected) throws IOException {
+
+        ArgumentsBuilder args = new ArgumentsBuilder();
+        args.add("--" + StandardArgumentDefinitions.INPUT_LONG_NAME);
```

Comment:
why not use args.addArgument and leave out the "--" or use args.addInput?

Reply:
Well for one because that didn't exist until I just jadded it, only the two argument version existed.